### ISSUE TITLE
breakpoint() not defined

### ISSUE THREAD

**Issue by jerryc05** [microsoft/pyright#123]
**Describe the bug**
pyright said breakpoint() function is not defined, but it actually is add in PEP553

**To Reproduce**
just write `breakpoint()`

**Expected behavior**
No error

**Screenshots or Code**
```
breakpoint()
```

**VS Code extension or command-line**
running pyright as a VS Code extension
Which version? 1.0.26

**Additional context**

**Comment by erictraut** [comment]
The breakpoint() function was added in Python 3.7. If you have Pyright configured to assume Python < 3.7, breakpoint will not be available. It is defined conditionally in the builtins.pyi file:

```
if sys.version_info >= (3, 7):
    def breakpoint(*args: Any, **kws: Any) -> None: ...
```

Have you set the "pythonVersion" setting in pyrightconfig.json to a value below "3.7"?

**Comment by jerryc05** [comment]
@erictraut Hi I cannot find pyrightconfig.json anywhere in my PC. I found something like `pyrightconfig.schema.json`, and pythonVersion is set to 3.7 in that file.

**Comment by erictraut** [comment]
Ah yes, you're right. It is a bug in pyright. I'll have a fix in the next version. Thanks for the bug report!

**Comment by erictraut** [comment]
This is now fixed in 1.0.27, which I just published.
